index.html

```diff
@@ -1212,9 +1221,18 @@
       <a>NDEF record</a> types.
     </p>
     <p>
-      The <dfn>NDEFRecord.data()</dfn> method returns the
-      payload data of the <a>NDEF record</a> with an appropriate [[!ECMAScript]]
-      type, which depends on the <a>MIME type</a>.
+      The <dfn>NDEFRecord.toText()</dfn> method, when invoked, MUST return the
+      result of running <a>convert an NDEFRecord object's bytes</a> with <i>text</i>,
+      unless there's other specific illustration in step 7 of
+      <a href="#the-nfc-reading-algorithm">The NFC reading algorithm</a>.
+    </p>
+    <p>
+      The <dfn>NDEFRecord.toArrayBuffer()</dfn> method, when invoked, MUST return the
+      result of running <a>convert an NDEFRecord object's bytes</a> with <i>arrayBuffer</i>.
+    </p>
+    <p>
+      The <dfn>NDEFRecord.toJSON()</dfn> method, when invoked, MUST return the
```

Comment:
```suggestion
      The <dfn>toJSON()</dfn> method, when invoked, MUST return the
```

And add `data-dfn-for="NDEFRecord"` above somewhere.

Reply:
Thank you! Lesson learned!

Reply:
Done.